packages/tools/changelog-generator-wrapper/package.json

```diff
@@ -33,8 +33,8 @@
 		"tsc": "echo \"No build required.\""
 	},
 	"dependencies": {
-		"@changesets/cli": "^2.26.1",
-		"@changesets/types": "^5.2.1",
+		"@changesets/cli": "^2.27.8",
+		"@changesets/types": "^6.0.0",
```

Comment:
Not sure this is safe. There is interplay between this package and build-tools. Do the release notes say anything about why it's a major release?

Reply:
It's due to this change, which I don't think should matter to us?
https://github.com/changesets/changesets/pull/1185#issuecomment-1621133148

(was originally checked in as minor, then readjusted to major):
https://github.com/changesets/changesets/pull/1186

https://github.com/changesets/changesets/releases/tag/%40changesets%2Ftypes%406.0.0

Reply:
Ha, yeah, that shouldn't affect us because we have modernized our build-env already. Yay!